bindata/network/openshift-sdn/002-rbac.yaml

```diff
@@ -58,31 +58,11 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: sdn
+  name: openshift-sdn
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: sdn
-subjects:
-- kind: ServiceAccount
-  name: sdn
-  namespace: openshift-sdn
-
----
-# Rather than PATCHing the SecurityContextConstraints object, we can bind to 
-# it. This is undocumented, thanks to evb who showed me this trick
-# TODO: When the openshift-api control plane is splut out, we will need to
-# remove this and replace it with something (sigh) hardcoded
-apiVersion: authorization.openshift.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: sdn-privileged
```

Comment:
what's up with the changes here?

Reply:
These are no longer needed because we are running in a special mode where we no longer need access to the privileged SecurityContextConstraint.